src/Uno.UI/Microsoft/UI/Xaml/Controls/TeachingTip/TeachingTip.cs

```diff
@@ -0,0 +1,2565 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Numerics;
+using Microsoft.UI.Private.Controls;
+using Microsoft.UI.Xaml.Automation.Peers;
+using Uno.Extensions;
+using Uno.UI.Helpers.WinUI;
+using Windows.ApplicationModel;
+using Windows.Devices.Sensors;
+using Windows.Foundation;
+using Windows.Graphics.Display;
+using Windows.System;
+using Windows.UI.Composition;
+using Windows.UI.Core;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Automation;
+using Windows.UI.Xaml.Automation.Peers;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Shapes;
+
+namespace Microsoft.UI.Xaml.Controls
+{
+	public partial class TeachingTip : ContentControl
+	{
+		private long m_automationNameChangedRevoker;
+		private long m_automationIdChangedRevoker;
+
+		public TeachingTip()
+		{
+			//__RP_Marker_ClassById(RuntimeProfiler.ProfId_TeachingTip);
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6388179)